src/logic/operate.js

```diff
@@ -0,0 +1,34 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  let total = 0;
+
+  switch (operation) {
+    case '%':
+      total = num1 % num2;
+      break;
+
+    case '÷':
+      total = num1 / num2;
+      break;
+
+    case '*':
+      total = num1 * num2;
+      break;
+
+    case '-':
+      total = num1 - num2;
+      break;
+
+    case '+':
+      total = num1 + num2;
+      break;
+    default:
+      total = 'Error: Cannot Perform operation';
```

Comment:
- [ ] Please use big.js operations here. Eg: `num1.plus(num2)`
- [ ] The percent(%) operator is not a modulus operator so rather divide the number by 100 instead.